go-controller/pkg/ovn/pods.go

```diff
@@ -69,6 +211,25 @@ func (oc *Controller) syncPods(pods []interface{}) {
 	}
 }
 
+// sync pod handler brings the pod resource in sync
+func (oc *Controller) syncPodHandler(key string) error {
+	namespace, name, err := cache.SplitMetaNamespaceKey(key)
+	if err != nil {
+		return err
+	}
+	pod, err := oc.pods.Pods(namespace).Get(name)
+	// It´s unlikely that we have an error different that "Not Found Object"
+	// because we are getting the object from the informer´s cache
+	if err != nil && !apierrors.IsNotFound(err) {
+		return err
+	}
+
```

Comment:
I don't know this logic and maybe this comment is redundant, but if you want to do a different action on delete, you can just check here
```go
if err != nil && apierrors.IsNotFound(err) {
 // do delete pod stuff, we receive the sync key and is not in the cache, so it has to be removed
}
```

Reply:
good catch. this was copy/pasted from your controller 😂 
I don't think i'd completely wired up deletes yet because I got stuck with the exGW stuff not being idempotent

Reply:
:) :100: